app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t "title"
```

Comment:
chỗ này em có thể lược bỏ `I18n` nhé, viết như này là được rùi:
```suggestion
    base_title = t "title"
```